dbt_coding_conventions.md

```diff
@@ -125,6 +125,15 @@ Projects should have a directory structure per the following example.
 
 <br>
 
+### What models should and shouldn’t be referenced by other models? dbt: ref()
```

Comment:
I would place this after the `#### source models` paragraph, maybe at H4 so it's inside the ### marts directory section

Reply:
Updated to suggestion, please let me know how that looks